.swcrc

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://swc.rs/schema.json",
+  "exclude": [".*\\.spec|test\\.(j|t)s$", "fixtures", "__tests__"],
+  "jsc": {
+    "parser": {
+      "syntax": "typescript",
+      "tsx": true
+    },
+    "target": "es2017",
```

Comment:
CSM needs to be at least this low for it to work with the contentful.js package. I didn't think it would be worth the extra config to target a different version for live preview only

Reply:
Ah right fair enough, cheers